doc/source/installing.rst

```diff
@@ -407,7 +407,7 @@ development version of yt instead of the latest stable release, you will need
   $ git clone https://github.com/yt-project/yt
   $ cd yt
   $ git checkout master
-  $ pip install -e . --user --prefix=
+  $ pip install . --user --prefix=
```

Comment:
``--user`` doesn't need `--install-option`, it's supported by pip: http://pip.readthedocs.io/en/stable/reference/pip_install/#cmdoption-user